### PR TITLE
Fix password policy translations

### PR DESCRIPTION
**Pull request by WoodySlum**
Define new constants to translate password policy rules

### CODE REVIEW

File: UI/PreferencesUI/UIxPreferences.m [hunk 1]
```diff
@@ -259,7 +260,16 @@ - (NSArray *) passwordPolicy
   NSObject <SOGoSource> *userSource;
 
   userSource = [user authenticationSource];
-  return [userSource userPasswordPolicy];
+  NSMutableDictionary *translations = [[NSMutableDictionary alloc] init];
```
Comment (cgx) [inline UI/PreferencesUI/UIxPreferences.m:263]: Keep variables declarations at the top of the current block (in this case, at the beginning of the method).

Comment (WoodySlum) [reply UI/PreferencesUI/UIxPreferences.m:261]: `translations` var is now on the top of the method, below `userSource`

Comment (cgx) [reply UI/PreferencesUI/UIxPreferences.m:261]: Please do the same with `policy` and `translatedUserPolicy`. Thanks!

Comment (WoodySlum) [reply UI/PreferencesUI/UIxPreferences.m:261]: done ;)

**Pull request merged**